apps/sharebymail/lib/ShareByMailProvider.php

```diff
@@ -839,7 +839,7 @@ public function getSharesBy($userId, $shareType, $node, $reshares, $limit, $offs
 					$or1
 				)
 			);
-		} else {
+		} else if ($node === null) {
```

Comment:
```suggestion
		} elseif ($node === null) {
```
It is really weird that this passes the codesniffer ci

Reply:
Done